.circleci/config.yml

```diff
@@ -67,9 +30,5 @@ workflows:
   version: 2
   build:
     jobs:
-      - lint
```

Comment:
Removed because they are redundant with Kokoro. I still want to keep a few sessions around until I can get some nox sessions that install their dependencies with Conda.

Reply:
https://github.com/googleapis/python-bigquery-pandas/issues/398